erdos/utils.py

```diff
@@ -76,6 +77,8 @@ def setup_logging(name, log_file=None):
     if log_file is None:
         handler = logging.StreamHandler()
     else:
+        if '/' in log_file:  # Slash will be mistaken as path path in logger
```

Comment:
I think this is dangerous behavior because we're changing log_file implicitly. IMHO, we should provide an auxiliary method that developers can use if they want (e.g., `escape_log_file`), and raise an exception here if the log_file is not suitable.